test_conformance/math_brute_force/binary_two_results_i_double.cpp

```diff
@@ -147,7 +147,16 @@ int TestFunc_DoubleI_Double_Double(const Func *f, MTdata d, bool relaxedMode)
         for (auto j = gMinVectorSizeIndex; j < gMaxVectorSizeIndex; j++)
         {
             uint32_t pattern = 0xffffdead;
-            memset_pattern4(gOut[j], &pattern, BUFFER_SIZE);
+
+            if ((error = clEnqueueFillBuffer(gQueue, gOutBuffer[j],
+                                             &pattern, sizeof(pattern), 0,
+                                             BUFFER_SIZE, 0, NULL, NULL)))
```

Comment:
As before, I think the subsequent call to clEnqueueWriteBuffer should be removed now because the initialization is being performed via clEnqueueFillBuffer instead, correct?